vehicle/renault.go

```diff
@@ -458,3 +475,10 @@ func (v *Renault) Climater() (active bool, outsideTemp float64, targetTemp float
 
 	return false, 0, 0, err
 }
+
+var _ api.VehicleStartCharge = (*Renault)(nil)
+
+// StartCharge implements the api.VehicleClimater interface
```

Comment:
`VehicleClimater` wirklich?

Reply:
Ja, diese Funktion gibt es in der App. Danach wird die Klimatisierung für 10 Minuten gestartet. Allerdings nur wenn Akku bei 40%> ist. Ob man das hier braucht ist halt die Frage. Denke dass man dafür eher die Dacia App verwendet und halt auch unterwegs nutzt.